src/lib/expression/aggregate_expression.hpp

```diff
@@ -4,7 +4,12 @@
 
 namespace opossum {
 
-enum class AggregateFunction { Min, Max, Sum, Avg, Count, CountDistinct, StandardDeviationSample };
+/**
+ * Supported aggregate functions. In addition to the default SQL functions (e.g., MIN(), MAX()), Hyrise internally uses
+ * the ANY() function, which return any single value. This function is only used by the optimizer in case that the
+ * group to aggregate is either of size one or all values of the group are known to be equal.
```

Comment:
Yeah, that's rather implicit for PKs. Changed to "This function is only used by the optimizer in case that all values of the group are known to be equal."